src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/DiagnosticsTelemetryModule.cs

```diff
@@ -80,6 +80,17 @@ public TimeSpan HeartbeatInterval
             set => this.HeartbeatProvider.HeartbeatInterval = value;
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether or not to query for and include Azure Instance Metadata in the
+        /// heartbeat properties. For more information on the Azure Instance Metadata service in Azure see
+        /// <a href="https://go.microsoft.com/fwlink/?linkid=864683">this document to learn more.</a>
+        /// </summary>
+        public bool EnableInstanceMetadata
```

Comment:
EnableAzureInstanceMetadata as name to make it obvious.

minor: the summary comment - remove the 'to learn more' at end as its redundant.